locales/en.default.json

```diff
@@ -103,6 +103,9 @@
   "products": {
     "product": {
       "add_to_cart": "Add to cart",
+      "added_to_cart": "Added to cart",
```

Comment:
This was the previous success message that was displayed on the button.

I won't remove it just yet because we were still discussing if we should bring it back, but good catch to remove before merging (and let the translation-platform take care of removing it later from other languages).